pyspeckit/spectrum/models/ammonia.py

```diff
@@ -204,6 +208,28 @@ def ammonia(xarr, trot=20, tex=None, ntot=14, width=1, xoff_v=0.0,
 
     return model_spectrum
 
+def cold_ammonia(xarr, tkin, **kwargs):
```

Comment:
(it can still be specified as a kwarg and will be whenever used by pyspeckit models)